tests/exporters/prometheus_exporter_test.py

```diff
@@ -81,6 +88,89 @@ def test_manage_gauge(self, mock_logger, mock_core):
         mock_logger.assert_called_once_with('%s \n', 'samples')
         assert metric_obj == mock_gauge_instance
 
+    @mock.patch('hanadb_exporter.exporters.prometheus_exporter.core')
+    @mock.patch('logging.Logger.error')
+    def test_incorrect_label(self, mock_logger, mock_core):
+
+        mock_gauge_instance = mock.Mock()
+        mock_gauge_instance.samples = 'samples'
+        mock_core.GaugeMetricFamily = mock.Mock()
+        mock_core.GaugeMetricFamily.return_value = mock_gauge_instance
+
+        mock_metric = mock.Mock()
+        mock_metric.name = 'name'
+        mock_metric.description = 'description'
+        mock_metric.labels = ['column4', 'column5']
+        mock_metric.unit = 'mb'
+        mock_metric.value = 'column3'
+
+        formatted_query = [
+            {'column1': 'data1', 'column2': 'data2', 'column3': 'data3'},
+            {'column1': 'data4', 'column2': 'data5', 'column3': 'data6'},
+            {'column1': 'data7', 'column2': 'data8', 'column3': 'data9'}
+        ]
+
+        with pytest.raises(ValueError) as err:
+            metric_obj = self._collector._manage_gauge(mock_metric, formatted_query)
+
+        assert('One or more label(s) specified in metrics.json'
```

Comment:
I would say this assert is incorrect, you are not asserting the `in` statement as it is outside the assert. It should be:
```
assert('One or more label(s) specified in metrics.json'
            ' for metric: "{}" is not found in the the query result'.format(
             'name') in str(err.value)) 
```